### PR TITLE
adding intitial setup for App component

### PR DESCRIPTION
**Pull request by KStockton**
Setting up App component

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,33 +1,47 @@
...
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.search-bar {
```
Comment (lboyer4) [inline src/App.css]: move this in to search-bar.css

File: src/App.jsx [hunk 2]
```diff
@@ -1,24 +1,39 @@
...
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
+          <h1>Climb On</h1>
```
Comment (lboyer4) [inline src/App.jsx:25]: Change to its own component

**Review by lboyer4 — CHANGES_REQUESTED** [review]
Add two comments

**Review by jarrettkong — APPROVED** [review]
Nice job breaking this into different components

**Pull request merged**